packages/cli/src/commands/dev-contracts.ts

```diff
@@ -16,6 +16,7 @@ const devOptions = {
   rpc: deployOptions.rpc,
   configPath: deployOptions.configPath,
   alwaysRunPostDeploy: deployOptions.alwaysRunPostDeploy,
+  postDeployForgeOptions: deployOptions.postDeployForgeOptions,
```

Comment:
what about just `forgeOptions` to keep it the same as the other commands?

Reply:
if not, kinda wondering about `forgeScriptOptions` since I am kinda anticipating more than just post deploy in the future: https://github.com/latticexyz/mud/issues/1755

Reply:
> if not, kinda wondering about `forgeScriptOptions` since I am kinda anticipating more than just post deploy in the future: #1755

Sure, seems like a better name, updated